examples/speech_to_text/whisper_waitk.py

```diff
@@ -0,0 +1,68 @@
+from typing import Optional
+from simuleval.agents.states import AgentStates
+from simuleval.utils import entrypoint
+from simuleval.data.segments import SpeechSegment
+from simuleval.agents import SpeechToTextAgent
+from simuleval.agents.actions import WriteAction, ReadAction
+
+import whisper
+import numpy
+
+
+@entrypoint
+class WaitkWhipser(SpeechToTextAgent):
```

Comment:
Nit: `Whipser` -> `Whisper` typo. Ditto with `eval_whipser_waitk.sh`, and in the README